documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -6,20 +9,2563 @@
+
+ifdef::community[]
+|[[mysql-property-gtid-new-channel-position]]<<mysql-property-gtid-new-channel-position, `gtid.new.channel.position`>> +
+_deprecated and scheduled for removal_
+|`earliest`
+|When set to `latest`, when the connector sees a new GTID channel, it starts consuming from the last executed transaction in that GTID channel. If set to `earliest` (default), the connector starts reading that channel from the first available (not purged) GTID position. `earliest` is useful when you have an active-passive MySQL setup where {prodname} is connected to the primary server. In this case, during failover, the replica with the new UUID (and GTID channel) starts receiving writes before {prodname} is connected. These writes would be lost when using `latest`.
+endif::community[]
+
+|[[mysql-property-tombstones-on-delete]]<<mysql-property-tombstones-on-delete, `tombstones.on.delete`>>
+|`true`
+|Controls whether a delete event is followed by a tombstone event. +
+ +
+`true`  - a delete operation is represented by a delete event and a subsequent tombstone event.  +
+ +
+`false` - only a delete event is emitted. +
+ +
+After a source record is deleted, emitting a tombstone event (the default behavior) allows Kafka to completely delete all events that pertain to the key of the deleted row. 
+
+|[[mysql-property-message-key-columns]]<<mysql-property-message-key-columns, `message.key.columns`>>
+|_empty string_
+|A semicolon separated list of tables with regular expressions that match table column names. The connector maps values in matching columns to key fields in change event records that it sends to Kafka topics. This is useful when a table does not have a primary key, or when you want to order change event records in a Kafka topic according to a field that is not a primary key. +
+ +
+Separate entries with semicolons. Insert a colon between the fully-qualified table name and its regular expression. The format (shown with spaces for clarity only) is: +
+ +
+_database-name_ `.` _table-name_ `:` _regexp_ `;` ... +
+ +
+For example: +
+ +
+`dbA.table_a:regex_1;dbB.table_b:regex_2;dbC.table_c:regex_3` +
+ +
+If `table_a` has an `id` column, and `regex_1` is `^i` (matches any column that starts with `i`), the connector maps the value in the `id` column of `table_a` to a key field in change events that the connector sends to Kafka. 
+
+|[[mysql-property-binary-handling-mode]]<<mysql-property-binary-handling-mode,`binary.handling.mode`>>
+|bytes
+|Specifies how binary columns, for example, `blob`, `binary`, `varbinary`, should be represented in change events. Possible settings:  +
+ +
+`bytes` represents binary data as a byte array. +
+ +
+`base64` represents binary data as a base64-encoded String. +
+ +
+`hex` represents binary data as a hex-encoded (base16) String.
+
+|===
+
+[id="mysql-advanced-connector-configuration-properties"]
+.Advanced MySQL connector configuration properties
```

Comment:
I am curious if rendering of this type of layout needs some CSS changes upstream.  What I actually see upstream when this gets rendered is the following:

```
Advanced MySQL connector configuration properties
The following table describes ...
```

You'll notice there is no spacing between the heading, no underline which essentially makes the heading flow into the text so you might actually miss notion that the section has changed to something else.  I've added [DBZ-2631](https://issues.redhat.com/browse/DBZ-2631) as a follow-up for this.

Reply:
DBZ-2631 has been resolved.

Reply:
Oh, great! I meant to bring this up. I don't see how you changed the rendering, but yes, we needed to somehow distinguish the 

`.This is a section title `

headings. So thanks for taking care of that!

Reply:
This is handled automatically by our CSS used by Antora in our website build process; so this is entirely transparent to your PR and only deals specifically with the upstream HTML generation step.